modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -282,8 +282,8 @@ def update_status!(status:, code: nil, detail: nil)
       update_status(status:, code:, detail:, raise_on_error: true)
     end
 
-    def submit_evidence_to_central_mail
-      evidence_submissions&.each(&:submit_to_central_mail)
+    def submit_evidence_to_central_mail!
+      evidence_submissions&.each(&:submit_to_central_mail!)
```

Comment:
Wanted to distinguish these methods from others in that they make changes to the underlying db record (i.e. use caution if calling directly).

Reply:
Beautiful